### PR TITLE
docs: Update readme for downloaded packages

### PR DESCRIPTION
**Pull request by xsrvmy**
Change the reference to VS Marketplace to GH repos, since the process upstream has changed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,17 +117,18 @@ This repo exists so that you don't have to download+build from source. The build
...
-  - [ms-vscode.vscode-js-profile-table](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-js-profile-table)
+- Extensions from Microsoft's GitHub repos (see product.json in vscode repo):
+  - ms-vscode.js-debug-companion
+  - ms-vscode.js-debug
+  - ms-vscode.vscode-js-profile-table
 - From [Electron releases](https://github.com/electron/electron/releases) (using [gulp-atom-electron](https://github.com/joaomoreno/gulp-atom-electron))
   - electron
   - ffmpeg
 
+The build process used to (but no longer) download built-in extensions from the VS Marketplace.
```
Comment (daiyam) [inline README.md:130]: I'm sure from where you are getting that.

Comment (xsrvmy) [reply README.md:130]: The PR I linked changed the download from marketplace to GitHub as a fix upstream. There was a discussion for this about how building vscode oss violated marketplace tos.

**Comment by daiyam** [comment]
In the current state, the PR isn't true since the 4 extensions are still downloaded from the marketplace. 
But it reveals that it can be done. 

Currently we have:
```
[01:17:09] Starting bundle-marketplace-extensions-build ...
[01:17:09] Downloading extension: ms-vscode.references-view@0.0.89 ...
[01:17:09] Downloading extension: ms-vscode.js-debug-companion@1.0.18 ...
[01:17:09] Downloading extension: ms-vscode.js-debug@1.67.2 ...
[01:17:09] Downloading extension: ms-vscode.vscode-js-profile-table@1.0.1 ...
[01:17:14] Finished bundle-marketplace-extensions-build after 4576 ms
```

We should have something like:
```
[01:17:09] Starting bundle-marketplace-extensions-build ...
[01:17:09] Downloading extension from GH: ms-vscode.references-view@0.0.89 ...
[01:17:09] Downloading extension from GH: ms-vscode.js-debug-companion@1.0.18 ...
[01:17:09] Downloading extension from GH: ms-vscode.js-debug@1.67.2 ...
[01:17:09] Downloading extension from GH: ms-vscode.vscode-js-profile-table@1.0.1 ...
[01:17:14] Finished bundle-marketplace-extensions-build after 4576 ms
```

**Comment by daiyam** [comment]
@xsrvmy If you could make another PR, the make the changes. It would be appreciated. Thx

**Comment by xaberus** [comment]
@daiyam This [line](https://github.com/microsoft/vscode/blob/main/build/gulpfile.extensions.js#L226) has to be patched. The `product.extensionsGallery?.serviceUrl` argument to `packageMarketplaceExtensionsStream` has to be removed. If no gallery link is provided, the GitHub code is used.
A new patch is needed, and I am struggling to understand how you manage the vscode patches? Is this described somewhere? Do you have a local tree that you rebase to upstream and keep the patches in there? Or do you regenerate them manually?

**Comment by daiyam** [comment]
@xaberus You can find my patching process here: https://github.com/VSCodium/vscodium/discussions/1053#discussioncomment-2593658
I will need to put that in a doc https://github.com/VSCodium/vscodium/blob/master/docs/build.md

**Comment by GitMensch** [comment]
Am I right that this PR is outdated and what we need for the docs is:

* document the patching process better @daiyam 
* adjust the current docs minimal (actually this PR) @xaberus

Correct?

**Pull request closed without merging**